### PR TITLE
Bug Fixes

### LINKED ISSUE #319: Show custom specie image on specie number selection
**Describe the bug**
The custom image for a specie is not used when choosing the amount of trees registered for this specie.

**To Reproduce**
1. Register multiple trees, e.g. only one offsite point 
2. See error

**Screenshots**
Screenshot_1620121357Screenshot_1620121379

**Smartphone (please complete the following information):**
- Device: Android emulator
- OS: Android 11
- Tree Mapper App Version: 1.0.1

### PR DESCRIPTION
**Pull request by tejassonar**
Fixes #319 , #280 

Changes in this pull request:
- Made multiple tree selection default
- changed sample tree count option to 5,10,20 and custom
- Tree tag by default enabled
- title for the sample tree map marking heading changed. e.g. (Sample Tree #1 of 10)
- moved location to last in the overview screen
- clicking on species_name on manage species should open species info.
- take a picture on specieInfo Screen, (fix UI, back button etc)
- Tree Alias should be used in the Manage Species screen.
- Delete options for multiple trees (for not uploaded)
- showing Verify Email Alert if email isn't verified

### CODE REVIEW

**Comment by norbertschuler** [comment]
- [ ] Bug #319 does not seem to be fixed. I still see the same image when asked about the number of planted specie after having done a multiple tree registration like this:

Simulator Screen Shot - iPhone 11 - 2021-05-17 at 16 14 40

**Comment by norbertschuler** [comment]
Choosing custom sample trees has some issues

- [ ] The area to insert a custom number of trees is quite small on iOS and it can only be edited if clicking on the small input area field while clicking the rest of the custom button area does nothing - it would be nice if clicking anywhere in the custom area would activate the input field and the custom tree option.
- [ ] When selecting the custom input field the formerly selected option sometimes stays selected (two areas are marked green) and despite entering a custom number if you click "continue" you have to register the number of sample trees choosen from the fixed 5,10 or 20 trees instead of the custom number of trees. This happens if you once enter a number, then choose a fixed amount again and then again choose the custom number:

Simulator Screen Shot - iPhone 11 - 2021-05-17 at 16 20 49Simulator Screen Shot - iPhone 11 - 2021-05-17 at 16 29 46

**Comment by norbertschuler** [comment]
> * moved location to last in overview screen

Does this mean this pull request also fixes https://github.com/Plant-for-the-Planet-org/treemapper/issues/274?

**Comment by norbertschuler** [comment]
https://github.com/Plant-for-the-Planet-org/treemapper/issues/312 also seems to be fixed with this pull request, right?

File: app/components/SampleTrees/SampleTreesCount.js [hunk 1]
```diff
@@ -42,14 +49,47 @@ const TreeNumberSelection = ({ sampleTreesCount, selectedTreeCount, setSelectedT
...
+            paddingVertical: 0,
+            color: 'white',
+            alignSelf: 'center',
+            // marginBottom: 4,
+          }}
+          selectionColor={'white'}
+          keyboardType={'numeric'}
+          onFocus={() => setIsCustomSelected(true)}
+          textAlign={'center'}
+          onChangeText={(text) => {
```
Comment (norbertschuler) [inline app/components/SampleTrees/SampleTreesCount.js:70]: Not sure if there already is some validation of the input, but it should be limited to numbers >0 and maybe some upper limit?

**Review by norbertschuler — CHANGES_REQUESTED** [review]

**Comment by tejassonar** [comment]
> * [ ]  Bug #319 does not seem to be fixed. I still see the same image when asked about the number of planted specie after having done a multiple tree registration like this if the specie has no image (it works if the specie has a custom image) - so please use the same default image we use for the inventory list here:
> 
> Simulator Screen Shot - iPhone 11 - 2021-05-17 at 16 14 40Simulator Screen Shot - iPhone 11 - 2021-05-17 at 16 17 23

Oh yes. I forgot to change the default image there. I will do that.

**Comment by tejassonar** [comment]
> > * moved location to last in overview screen
> 
> * [ ]  Does this mean this pull request also fixes #274?

No. It means
image

**Comment by tejassonar** [comment]
> * [ ]  #312 also seems to be fixed with this pull request, right?

I was never able to reproduce that issue. Let me know If you can reproduce it. Else, maybe we can close that issue.

File: app/components/SampleTrees/SampleTreesCount.js (hunk 1 above)
Comment (tejassonar) [reply app/components/SampleTrees/SampleTreesCount.js:70]: Yeah. I will add the validation too.

**Comment by norbertschuler** [comment]
> > * [ ]   #312 also seems to be fixed with this pull request, right?
> 
> I was never able to reproduce that issue. Let me know If you can reproduce it. Else, maybe we can close that issue.

I tested and there was no error for me. That's why I thought you fixed it. I will tell if I ever can reproduce it :-)

**Review by norbertschuler — APPROVED** [review]
LGFM

**Pull request merged**